core/skeleton.py

```diff
@@ -31,7 +31,7 @@ class MetaBaseSkel(type):
 	_skelCache = {}  # Mapping kindName -> SkelCls
 	_allSkelClasses = set()  # list of all known skeleton classes (including Ref and Mail-Skels)
 
-	__reservedKeywords_ = {"self", "cursor", "amount", "orderby", "orderdir",
+	__reservedKeywords_ = {"self", "cursor", "amount", "orderby", "orderdir", "limit"
```

Comment:
I think this should better replace `"amount"` instead of be a new element?

Reply:
Good point. I think we should unblock amount later on (3.1?) - for now let's leave it there to ensure all projects migrate correctly.

Reply:
I see no reason to keep it blocked. "amount" could not be used before, if you unblock it now nothing will break.
The critical part is that "limit" is now blocked. But that is a project related problem...